src/components/view-switcher/ViewSwitcher.jsx

```diff
@@ -0,0 +1,40 @@
+import { useState } from 'react'
+import ToggleButton from '@mui/material/ToggleButton'
+
+import FormatListBulletedIcon from '@mui/icons-material/FormatListBulleted'
+import GridViewIcon from '@mui/icons-material/GridView'
+
+import { cardsViews } from '~/constants'
+import { styles } from './ViewSwitcher.styles'
+
+const ViewSwitcher = ({ setOffersView }) => {
+  const [cardsView, setCardsView ] = useState(cardsViews.inline)
+
+  const changeCardsView = (_event,view) => {
+    setOffersView(view)
+    setCardsView(view)
+  }
+
+  return (
+    <>
+      <ToggleButton
+        aria-label='inline card view' onClick={ changeCardsView }
+        selected={ cardsView === cardsViews.inline }
+        sx={ { ...styles.toggleButton, marginRight:'8px' } }
```

Comment:
you can add this in styles file

Reply:
in this way style "marginRight:'8px'" will be in all buttons but i need only in one

Reply:
you can use variables and destructure in styles file , const toggleButton = {some styles} then in styles object use grid: toggleButton , inline : {...toggleButton, mr: '8px'}

Reply:
refactored it